server/room.ts

```diff
@@ -271,13 +410,13 @@ export class Room {
     };
   }
 
-  async resetRoomVM() {
+  async stopVBrowser() {
```

Comment:
one thing we can do that I haven't done yet is convert these to class properties instead of class methods so we get autobinding of `this`. Otherwise, it's very easy to introduce difficult-to-find bugs when invoking these from `socket.on` calls.

Might be better to do in a separate PR though.

Reply:
from what I know we can skip that for now by writing it this way:

```
// this will be autobinded in here
stopVBrowser = async () => {}
```

Reply:
Yes, that's the class property syntax that autobinds.

It's not supported in the version of Node I was using, which is why it was this why pre-TS.

Reply:
ah understand that! shouldn't be a problem anymore right?

Reply:
Yeah should be fine to use that syntax now.